liblocation/src/test/java/com/mapbox/android/core/location/LocationEngineResultTest.java

```diff
@@ -36,6 +41,49 @@ public void passValidIntent() {
     assertThat(result.getLastLocation()).isSameAs(location);
   }
 
+  @Test
+  public void passNullLocation() {
```

Comment:
Normally is good to structure the tests following the triple A 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40 http://wiki.c2.com/?ArrangeActAssert
If the `A`rrange part gets too long we should create factory or builder methods. 

This applies to the rest of the tests 👇